css/values.css

```diff
@@ -0,0 +1,336 @@
+/*
+===================================
+===================================
+Our Values
+===================================
+===================================
+*/
+
+/* 
+===================================
+Section Styles (background, padding, etc.)
+===================================
+*/
+
+/* 
+===================================
+Hero
+===================================
+*/
+.hero {
```

Comment:
This `background` shorthand is overwriting the other background properties declared in `global.css`.

<img width="331" alt="Screen Shot 2024-01-11 at 3 21 33 PM" src="https://github.com/indigenousfriends/itlct-website-development/assets/704697/994aa438-3200-4b65-9e1f-b5ad7fd61e07">

Reply:
Also, let's use `background-position: center` instead of `background-position: top center`.